test/unit/uk/gov/hmrc/tariffclassificationfrontend/service/CasesService_SuppressCaseSpec.scala

```diff
@@ -57,34 +57,62 @@ class CasesService_SuppressCaseSpec extends UnitSpec with MockitoSugar with Befo
   "Suppress a Case" should {
     "update case status to SUPPRESSED" in {
       // Given
+      val existingAttachment = mock[Attachment]
+
+      val fileUpload = mock[FileUpload]
+      val fileUploaded = FileStoreAttachment("id", "email", "application/pdf", 0)
       val operator: Operator = Operator("operator-id", Some("Billy Bobbins"))
-      val originalCase = aCase.copy(status = CaseStatus.NEW)
+      val originalCase = aCase.copy(status = CaseStatus.NEW, attachments = Seq(existingAttachment))
       val caseUpdated = aCase.copy(status = CaseStatus.SUPPRESSED)
 
-      given(connector.updateCase(refEq(caseUpdated))(any[HeaderCarrier])).willReturn(successful(caseUpdated))
+      given(fileStoreService.upload(fileUpload)).willReturn(successful(fileUploaded))
+      given(connector.updateCase(any[Case])(any[HeaderCarrier])).willReturn(successful(caseUpdated))
       given(connector.createEvent(refEq(caseUpdated), any[NewEventRequest])(any[HeaderCarrier])).willReturn(successful(mock[Event]))
 
       // When Then
-      await(service.suppressCase(originalCase, operator)) shouldBe caseUpdated
+      await(service.suppressCase(originalCase, fileUpload, "note", operator)) shouldBe caseUpdated
 
       verify(audit).auditCaseSuppressed(refEq(originalCase), refEq(caseUpdated), refEq(operator))(any[HeaderCarrier])
 
       val caseUpdating = theCaseUpdating(connector)
       caseUpdating.status shouldBe CaseStatus.SUPPRESSED
+      caseUpdating.attachments should have(size(2))
+
+      val attachmentUpdating = caseUpdating.attachments.find(_.id == "id")
+      attachmentUpdating.map(_.id) shouldBe Some("id")
+      attachmentUpdating.map(_.public) shouldBe Some(false)
+      attachmentUpdating.flatMap(_.operator) shouldBe Some(operator)
 
       val eventCreated = theEventCreatedFor(connector, caseUpdated)
       eventCreated.operator shouldBe Operator("operator-id", Some("Billy Bobbins"))
       eventCreated.details shouldBe CaseStatusChange(CaseStatus.NEW, CaseStatus.SUPPRESSED)
     }
 
+    "update update on attachment upload failure" in {
```

Comment:
Is this test unfinished? The name of it makes no sense, and nothing is being asserted.

Reply:
Not sure what happened there, sorted